superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/utils.ts

```diff
@@ -98,4 +98,4 @@ export const hasTemporalColumns = (
 export const doesColumnMatchFilterType = (filterType: string, column: Column) =>
   !column.type_generic ||
   !(filterType in FILTER_SUPPORTED_TYPES) ||
-  FILTER_SUPPORTED_TYPES[filterType].includes(column.type_generic);
+  FILTER_SUPPORTED_TYPES[filterType]?.includes(column.type_generic);
```

Comment:
nit: This change isn't necessary due to `!(filterType in FILTER_SUPPORTED_TYPES)` above (but I don't mind having it there if the logic changes later)

Reply:
This change was done because sometimes there were error with `EXPERIMENTAL_FEATURE` FF enabled